packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -86,24 +87,29 @@ export const UnitRow = ({
   revealSecondaryModal,
   alertBarRevealed,
   hideCtaText,
+  prefixDataTestId = '',
 }: UnitRowProps & RouteComponentProps) => {
   ctaText = ctaText || (headerValue ? 'Change' : 'Add');
 
   const location = useLocation();
   const multiButton = !!(route || secondaryCtaRoute);
 
+  function formatDataTestId(id: string) {
+    return prefixDataTestId ? `${prefixDataTestId}-${id}` : id;
+  }
+
   return (
     <div className="unit-row">
       <div className="unit-row-header">
         <span className="flex justify-between items-center">
-          <h3 data-testid="unit-row-header">{header}</h3>
+          <h3 data-testid={formatDataTestId('unit-row-header')}>{header}</h3>
```

Comment:
@jodyheavener You mentioned wanting to have intern use the `data-testid` to identify elements for testing. This mostly works, however in the `unit-row` and `input` components, if you have multiple of that component, it will complain that selector is not specific enough.

Using `prefixDataTestId` to prefix the component seems to solve this issue. Is there a better react way of doing this?

Reply:
Apologies, I actually don't recall my suggestion of that 🤔 But it was probably in reference to the second point of [this comment](https://github.com/testing-library/react-testing-library/issues/479#issuecomment-529548194). Mostly a convenience thing; if we have the test-ids there it would be handy to utilize them. We could certainly prefix them to de-dupe, I don't know of a more standardized method, though.

Reply:
@jodyheavener Sounds good, I'll stick with de-duping them.